README.md

```diff
@@ -1,106 +1,129 @@
+## Step 3: Define database and schema variables
+By default, this package runs using your destination and the `linkedin_ads` schema. If this is not where your Linkedin Ads data is (for example, if your linkedin schema is named `linkedin_ads_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
 # dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    linkedin_database: your_database_name
-    linkedin_schema: your_schema_name
+    linkedin_ads_database: your_destination_name
+    linkedin_ads_schema: your_schema_name
 ```
 
+## (Optional) Step 4: Additional configurations
+
 ### Switching to Local Currency
 Additionally, the package allows you to select whether you want to add in costs in USD or the local currency of the ad. By default, the package uses USD. If you would like to have costs in the local currency, add the following variable to your `dbt_project.yml` file:
 
 ```yml
 # dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    linkedin__use_local_currency: True
+    linkedin_ads__use_local_currency: True # false by default -- uses USD
 ```
 
 ### Passing Through Additional Metrics
-By default, this package will select `clicks`, `impressions`, and `costs` from the source `ad_analytics_by_creative` table to store into the staging model. If you would like to pass through additional metrics from this table to both the staging model and the final `linkedin__ad_adapter` transform model, add the following configuration to your `dbt_project.yml` file:
+By default, this package will select `clicks`, `impressions`, and `cost` from `ad_analytics_by_creative` and `ad_analytics_by_campaign`. If you would like to pass through additional metrics to the staging models, add the variable configuration in the code block below to your `dbt_project.yml` file. These variables allow for the pass-through fields to be aliased (`alias`) and casted (`transform_sql`) if desired, but not required. Datatype casting is configured via a sql snippet within the `transform_sql` key. You may add the desired sql while omitting the `as field_name` at the end and your custom pass-though fields will be casted accordingly. Use the below format for declaring the respective pass-through variables:
 
 ```yml
 # dbt_project.yml
-
-...
 vars:
-    linkedin__passthrough_metrics: ['the', 'list', 'of', 'metric', 'columns', 'to', 'include'] # from LINKEDIN_ADS.AD_ANALYTICS_BY_CREATIVE
+    linkedin_ads__passthrough_metrics: 
+        - name: "new_custom_field"
+          alias: "custom_field"
+        - name: "unique_int_field"
+          alias: "field_id"
+          transform_sql: "cast(field_id as int)"
+        - name: "that_field"
```

Comment:
Is it true that all passthrough columns will be available in both the `ad_analytics_by_campaign` and `ad_analytics_by_creative` source tables? I worry a user will want to passthrough a metric that exists in one source but not the other. Would it be better to simply have two variables for each table instead of the one?

Reply:
ack that's a really good catch -- it does look like we have columns in `ad_analytics_by_campaign` that are not in `ad_analytics_by_creative`. though curiously these non-overlapping fields are all averages

Reply:
made 2 distinct variables and updated documentation related to passthrough columns (changelog, readme)